src/components/react-admin/MapPointInput.js

```diff
@@ -34,10 +33,21 @@ const MapPointInput = ({ classes, input }) => {
     customProps.center = input.value;
   }
 
+  React.useEffect(() => {
+    if (mapConfig.accessToken) {
+      Map = ReactMapboxGl({
```

Comment:
Sorry for the repost, I still don't understand why we use React Mapbox Gl instead of `Map` component of terralego/core which is also a wrapper of Mapbox Gl and it's already packaged in this project.

Reply:
[Map component](https://github.com/Terralego/terra-front/blob/master/src/modules/Map/Map/Map.js) from [terra-front](https://github.com/Terralego/terra-front) does not allow _composition_ of MapboxGL elements. It only provides management for a custom configuration object which should provide all map content.

Current use case won't take any advantage of it.